MobileSDK/sdk/src/main/java/org/lantern/mobilesdk/Lantern.java

```diff
@@ -6,26 +6,17 @@
 import com.google.android.gms.analytics.HitBuilders;
 import com.google.android.gms.analytics.Tracker;
 
-
 import java.io.File;
 import java.util.HashMap;
 import java.util.Map;
-import java.util.concurrent.atomic.AtomicBoolean;
 
 /**
- * Created by ox.to.a.cart on 1/28/16.
+ * API for embedding the Lantern proxy
  */
-public class Lantern {
+public abstract class Lantern {
```

Comment:
The `enable()` method returns the address of the proxy.  Can you just use that from within Firetweet to configure the webview?